sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -45,7 +45,7 @@ import org.apache.spark.util.Utils
  * Spark also includes more built-in functions that are less common and are not defined here.
  * You can still access them (and all the functions defined here) using the `functions.expr()` API
  * and calling them through a SQL expression string. You can find the entire list of functions
- * at SQL API documentation.
+ * at SQL API documentation (https://spark.apache.org/docs/latest/api/sql/index.html).
```

Comment:
the problem is that the link became stale. For example, in Spark 3.0.0 release documentation, https://spark.apache.org/docs/latest/api/sql/index.html will point out the latest version of Spark.

Reply:
I tried to address this issue with an additional commit.